package.json

```diff
@@ -1,30 +1,25 @@
 {
   "name": "@gemeente-denhaag/denhaag-component-library",
   "version": "0.1.0",
+  "private": true,
   "description": "A library of React components for the Design System of The Netherlands",
-  "author": "Municipality of The Hague",
-  "license": "EUPL-1.2",
+  "bugs": "https://github.com/Gemeente-DenHaag/denhaag-component-library/issues",
   "repository": {
     "type": "git",
     "url": "https://github.com/Gemeente-DenHaag/denhaag-component-library.git"
   },
-  "bugs": "https://github.com/Gemeente-DenHaag/denhaag-component-library/issues",
-  "main": "dist/index.js",
-  "module": "dist/index.modern.js",
-  "source": "src/index.tsx",
+  "license": "EUPL-1.2",
+  "author": "Municipality of The Hague",
   "workspaces": [
     "src/components/*",
     "src/meta-packages/*",
     "src/"
   ],
-  "private": true,
-  "engines": {
-    "node": ">=10",
-    "yarn": "^1.22.0"
-  },
   "scripts": {
-    "build": "microbundle-crl --no-compress --format modern,cjs",
+    "bootstrap": "lerna bootstrap",
+    "build": "lerna run build",
     "build-storybook": "build-storybook",
+    "clean": "lerna run clean --parallel",
```

Comment:
Didn't the use of --parallel crash your machine @GewoonMaarten ?

Reply:
`clean` ran just fine, but `build` with `--parallel` crashed my laptop.